### ISSUE TITLE
Improve stability testing

### ISSUE THREAD

**Issue by sfogarty** [Trinity-Automata-Research/dsmodels#87]
Currently stability testing is far too sensitive to epsilon, tolerance, and iters. These are probably not the correct parameters for simbasins when testing stability, instead of rendering an image.

**Comment by sfogarty** [comment]
[ ] Figure out the right way to test stability in raw R.
[ ] Figure out the correct parmeters for simbasins
[ ] Update simbasins to match.

**Comment by kmclaren-trinity** [comment]
sim.is.stable should be updated to something like convergeTesting's is.stable

also, see issue #88

**Comment by kmclaren-trinity** [comment]
stable means "has a single attractor"
as a note, there may be a single attractor by every other point will be periodic around it. ONly that EXACT attractor will hit.

**Comment by kmclaren-trinity** [comment]
See #105

**Comment by kmclaren-trinity** [comment]
Sometimes we assume that a model is stable, and we want to know if the single stable point is interior or on the axes. That can be done by checking a single point.

Sometimes we want to check that the model IS stable (for all points).
